### ISSUE TITLE
On the potential deprecation of --glx-fshader-win 

### ISSUE THREAD

**Issue by NerveCoordinator** [chjj/compton#542]
> --glx-fshader-win is being deprecated, and might be removed in the future. If you really need this feature, please report an issue to let us know

I went looking for making parts of windows transparent based on colors, and [this](https://github.com/chjj/compton/issues/183) was the only thing I found that actually works. So as per the quote above that appears when using it, my preference is for the feature to remain supported.

**Comment by chrhasse** [comment]
I agree. Though I would prefer an easier to configure option that basically takes 1 or more colors in a given application, and maps them to other specified colors. That's essentially what I do with my shader right now, but having to write or customize a shader is a notably higher bar than editing a configuration file, and I can't specify that the mapping is for only one application.

**Comment by yshui** [comment]
This is a changed made by https://github.com/yshui/compton

Please refer to yshui/compton#108
